composer.json

```diff
@@ -22,8 +22,7 @@
   },
   "require-dev": {
     "codegyre/robo": "~0.4",
-    "phpunit/php-code-coverage": "~2.0",
-    "satooshi/php-coveralls": "~0.6"
+    "phpunit/php-code-coverage": "~2.0"
```

Comment:
Since you're changing the Composer config, it might be worthwhile to switch to proper [SemVer constraints](https://getcomposer.org/doc/articles/versions.md#caret) using a caret (`^`) while you're at it.